definitions/scenarios/maintenance_mode.rb

```diff
@@ -24,8 +28,11 @@ class MaintenanceModeStop < ForemanMaintain::Scenario
 
     def compose
       add_step(Procedures::Iptables::RemoveChain.new)
-      add_step(Procedures::Cron::Start.new)
-      add_step(Procedures::MaintenanceFile::Remove.new)
+      add_step(Procedures::SyncPlans::Enable.new)
+      if feature(:cron)
```

Comment:
While cron_service can only be `crond` it should be enough to
```ruby
add_step(Procedures::Service::Start.new(:only => 'crond')) if feature(:cron)
```
or do I miss something?